stats/analytics.py

```diff
@@ -149,6 +151,45 @@ def get_codelist_mapping(major_version):
         )
 
 
+def build_org_prefix_list():
+    """Build lists of valid organisation identifier prefixes"""
+    out = {}
+    for major_version in ('1', '2'):
+        out[major_version] = defaultdict(list)
+        for prefix in CODELISTS[major_version]['OrganisationRegistrationAgency']:
+            out[major_version][len(prefix)].append(prefix)
```

Comment:
What’s the thinking behind splitting the list by prefix length? Why not just make a big list of prefixes by version?

I think it’s okay either way, I’m mostly just curious.